go.mod

```diff
@@ -62,11 +62,11 @@ require (
 	github.com/mvdan/xurls v1.1.0
 	github.com/onsi/ginkgo v1.14.0
 	github.com/onsi/gomega v1.10.1
-	github.com/opencontainers/runc v1.0.3
+	github.com/opencontainers/runc v1.1.0
```

Comment:
I am not actually sure why this is getting bumped...

/shrug

Reply:
runc was bumped in cadvisor which is why this was getting picked up from this PR. Rebased on top of @kolyshkin's runc 1.1.0 patch in order to test.